ansible/files/runtimes.json

```diff
@@ -57,8 +57,15 @@
                 },
                 "stemCells": [
                     {
-                        "count": 2,
-                        "memory": "256 MB"
+                        "initialCount": 2,
```

Comment:
I think this would be a breaking change.
How about sharing this to the dev list as well?

Reply:
renaming the field would be a breaking change - better to support both as a way to allow backward compatibility. get(initialCount).orElse(count) for example.

Reply:
@rabbah ,i think we can change `count` to `initialCount` directly,

And your said problem: `renaming the field would be a breaking change`, i think it is not a problem, because this pr's all changes in invoker side only, after applied this pr to new invoker, has no bad influences on old invokers and other components(e.g. controller).

BTW, regrading `xxx.get(initialCount).orElse(count)`
What's `xxx` here? and `.get` is wroted where?

Reply:
@rabbah , already added custom json deserializer to support `initialCount` and `count` in runtimes.json